### PR TITLE
 internal/function: implement language function 

### LINKED ISSUE #192: implement UDF that gets the language from the content of the blob and/or its path

### PR DESCRIPTION
**Pull request by erizocosmico**
Closes #192 

Can be used as:

```sql
language(path)

OR

language(path, blob)
```

### CODE REVIEW

**Comment by erizocosmico** [comment]
Rebased

File: internal/function/language.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	return fn(&Language{left, right})
+}
+
+// Eval implements the Expression interface.
+func (f *Language) Eval(session sql.Session, row sql.Row) (interface{}, error) {
```
Comment (jfontan) [inline internal/function/language.go]: Eval now uses `sql.Context` instead of `Session`

Comment (erizocosmico) [reply internal/function/language.go]: Oh, right, the update is already on master

**Comment by erizocosmico** [comment]
Rebased

**Pull request merged**